test/e2e/e2e_test.go

```diff
@@ -464,6 +472,115 @@ var _ = ginkgo.Describe("Kueue", func() {
 		})
 	})
 
+	ginkgo.When("Suspend a running Job without requeueing through Workload's spec.active field", func() {
```

Comment:
Please move this case to integration test: https://github.com/kubernetes-sigs/kueue/blob/71b5cee8da608c1fa3bdd462641d53fa67f6bde5/test/integration/controller/jobs/job/job_controller_test.go

Reply:
Ohh so basically the whole e2e test should be moved `[integration/controller/jobs/job/job_controller_test.go]` ?

Reply:
My preference is also to make this an integration test, which gives us greater control.

integration tests are cheaper to run too.

Reply:
okok sounds good, i'll move it there!